### ISSUE TITLE
Add a limit to the description of projects because they are overlapping over the title

### ISSUE THREAD

**Issue by siddhant1** [ossn/fixme#108]

**Comment by siddhant1** [comment]
Maybe we can limit the word count of description and add a read more... to it

**Comment by alexdor** [comment]
Sounds good, this has to be done in a responsive way though : )

**Comment by fezheng397** [comment]
Hi, was just wondering if this is already fixed? I didn't seem to see a problem with it in the current projects page, but if it isn't, I'd be happy to work on the issue!

**Comment by siddhant1** [comment]
Sure , you can work on it !

**Comment by fezheng397** [comment]
Awesome! Would you have an example of the problem and what is preferred for a possible solution? If not, I can just fix it based on my intuition and send a few screenshots for feedback.

**Comment by siddhant1** [comment]
Yeah , you can play with your own styles , just make sure it's consistent with the overall theme :)

**Comment by siddhant1** [comment]
@fezheng397 are you still working on this one?

**Comment by fezheng397** [comment]
@siddhant1 So sorry for the late update and reply, I've had to deal with some personal issues lately and I apologize as I haven't been able to work on this. If possible, could you try reassigning this to someone else?

**Comment by siddhant1** [comment]
No problem @fezheng397 I will tackle this , thanks for help :)

**Comment by ebouJ** [comment]
@siddhant1 is this issue still open for contribution? Can I take on this?

**Comment by siddhant1** [comment]
Yes , go for it!

**Comment by idoqo** [comment]
Hi @siddhant1, is this still opened? 
I'm hoping to work with FixMe for this year's summer of code and was looking for good beginner-friendly issues to help me get familiar with the code base. 

Would be happy to take on this.

**Comment by siddhant1** [comment]
Yes its still open but i am not sure about the summer of code part , you can check mozilla forum to verify if the fixme is coming as a project (since organisations are not released yet)
